cg/meta/archive/ddn_dataflow.py

```diff
@@ -38,13 +42,18 @@ class ResponseFields(str, Enum):
     RETRIEVE_FILES = "files/retrieve"
 
 
-class TransferData(BaseModel):
+class DataFlowFileTransferData(FileTransferData):
```

Comment:
Please help with the naming... This is a model representing a single file and containing the information needed for the archiving, ie where the file is, any metadata and where to put it

Reply:
Some options:
- `DDNTransferData`
- `DDNArchiveData`

Reply:
Maybe the base model can be called `TransferData`, `TransferTask`. Naming is hard 😅

Reply:
I think in DataFlow is a bit superfluous here given the naming of the class. Maybe KarolinskaFileData, KarolinskaFile or FileData would be sufficient?

Reply:
```FileTransferInfo``` could also be applicable since it contains specifics as the where and what and not just data, but ```TransferData``` is sufficient also.

Reply:
After consulting with @islean we have decided to move away from using `DDN` or just `DataFlow`, and instead use Miria (actual software used). Here I then suggest MiriaFile and then refactor the rest in a separate PR